administrator-manual/en/antivirus.rst

```diff
@@ -0,0 +1,26 @@
+.. _antivirus-section:
+
+=========
+Antivirus
+=========
+
+.. note::
+
+  The configuration page of this module is available only in the new Server Manager.
+
+ClamAV is the open source antivirus engine of |product|. |product| uses 2 different ClamAV instances:
```

Comment:
```suggestion
ClamAV is the open source antivirus engine of |product|. There run two ClamAV instances:
```

Reply:
We run two...

Reply:
I don't like the "we" in this context.

I'm going with:
```
The server runs two ...
```